apps/site/pages/en/about/get-involved/index.md

```diff
@@ -16,11 +18,18 @@ layout: about
 - The official Node.js Twitter account is [nodejs](https://twitter.com/nodejs).
 - The [Node.js project calendar](https://nodejs.org/calendar) with all public team meetings.
 
-## Learning
+## Learning Materials
 
 - [Official Learn section](https://nodejs.org/en/learn/) of the Node.js website.
 - [Official API reference documentation](https://nodejs.org/api/).
 - [NodeSchool.io](https://nodeschool.io/) will teach you Node.js concepts via interactive command-line games.
 - [Stack Overflow Node.js tag](https://stackoverflow.com/questions/tagged/node.js) collects new information every day.
 - [The DEV Community Node.js tag](https://dev.to/t/node) is a place to share Node.js projects, articles and tutorials as well as start discussions and ask for feedback on Node.js-related topics. Developers of all skill-levels are welcome to take part.
+
+## Unofficial Discussion Areas
```

Comment:
Can the IRC and OpenJS Slack be moved here? Since they are both unofficial discussion platforms.

Reply:
The only "official" platform in th future for external community discussions besides this GitHub will be our Discord server (Nodeiflux transitioning)

Reply:
slack is no longer official ???

Reply:
OpenJS slack was never an official Node community

Reply:
Ok good to know so I I'll mode openjsf slack to unofficial but nodeiflux will stay on unofficial 'cause any decision was take